src/core/server/saved_objects/migrationsv2/integration_tests/actions.test.ts

```diff
@@ -207,15 +207,19 @@ describe('migration actions', () => {
   });
 
   describe('cloneIndex', () => {
-    afterEach(async () => {
+    afterAll(async () => {
       try {
-        await client.indices.delete({ index: 'yellow_then_green_index' });
+        await client.indices.delete({ index: 'clone_*' });
       } catch (e) {
         /** ignore */
       }
     });
     it('resolves right if cloning into a new target index', () => {
-      const task = cloneIndex(client, 'existing_index_with_write_block', 'yellow_then_green_index');
+      const task = cloneIndex(
+        client,
+        'existing_index_with_write_block',
+        'clone_yellow_then_green_index_1'
+      );
       expect(task()).resolves.toMatchInlineSnapshot(`
```

Comment:
we need to await `expect` when using `resolves` or `rejects`

Reply:
Good catch, this was probably the original cause of the flakiness! I hate that it's this easy to write a test that does nothing 👎 

I added `expect.assertions(N)` to all the tests as safety guard.